operators/src/processing/meteosat/radiance.rs

```diff
@@ -0,0 +1,502 @@
+use crate::engine::{
+    ExecutionContext, InitializedRasterOperator, Operator, QueryContext, QueryProcessor,
+    RasterOperator, RasterQueryProcessor, RasterQueryRectangle, RasterResultDescriptor,
+    SingleRasterSource, TypedRasterQueryProcessor,
+};
+use crate::util::Result;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::primitives::SpatialPartition2D;
+use geoengine_datatypes::raster::{
+    EmptyGrid, Grid2D, GridShapeAccess, Pixel, RasterDataType, RasterPropertiesKey, RasterTile2D,
+};
+use num_traits::AsPrimitive;
+use std::convert::TryFrom;
+
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+
+//TODO Check
+type PixelOut = f32;
```

Comment:
What do we need to check?

Reply:
Check whether output is always f32. :D
Removed in: 3be3fd383500c6069bb49ffde1523b83a9c4b184